can/interfaces/vector/canlib.py

```diff
@@ -380,5 +403,6 @@ def _detect_available_configs():
                      channel_config.name.decode('ascii'))
             configs.append({'interface': 'vector',
                             'app_name': None,
-                            'channel': channel_config.channelIndex})
+                            'channel': channel_config.channelIndex,
+                            'serialNumber': channel_config.serialNumber})
```

Comment:
See my comment above about creating a new method instead of re-using this one.

I've been thinking that it might be a good idea to create a new base class for representing channels in python-can. Then it could contain more useful information like name, article number, serial number etc in a backend independent manner.